radio/src/telemetry/frsky.cpp

```diff
@@ -283,7 +286,14 @@ void telemetryWakeup()
 {
 #if defined(CPUARM)
   uint8_t requiredTelemetryProtocol = MODEL_TELEMETRY_PROTOCOL();
+#if defined(REVX)
+  uint8_t requiredSerialInversion = g_model.moduleData[EXTERNAL_MODULE].invertedSerial;
+  if (telemetryProtocol != requiredTelemetryProtocol
```

Comment:
Argh, again my work habits showing through. (We have a rule that no line may be longer than screen width.)
Will try to improve my context switching.